### DISCUSSION TITLE
"Stack" field for table

### DISCUSSION THREAD

**Discussion by msnegurski** [filamentphp/filament#265] [Ideas]
It would be cool to have a "Stack" field.

Sometimes resource can have more attributes that could fit even wide table view, that should be shown. Also sometimes there are attributes that almost don't deserve a dedicated column, but would be a great addition to already existing column.

As example, let's say users can have paid subscription. Would be great to show plan, subscription status and next billing date on users table, but it's three columns, which is a waste.

Ability to stack several attributes in one column would be awesome.

**Comment by danharrin** [comment]
I agree! This would be powerful.
